scripts/index.js

```diff
@@ -0,0 +1,106 @@
+import BookList from '../modules/book.js';
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
```

Comment:
- [ ] The luxon.js library is not loading properly, hence the date is not displaying. Kindly ensure to fix this issue.

Loading error:
![image](https://user-images.githubusercontent.com/12745474/148839128-81607747-0af5-4cb7-9ee1-18d67b990eb8.png)

Reply:
Hello @rloterh.
First of all, thank you for your feedback. 
I put the node_modules folder inside of *.gitignore* file. I kindly ask you Do you think I should push the *node_modules* folder to github repository?. At the moment, I am not using a *webpack* or similar tool, that's the reason I can not import the library as *from 'luxon'*. Could you give some suggestions to solve this?
Thank you in advance.

Reply:
@gabyse1 
No, the node module needs not to be pushed. It should only remain in the .gitignore. 
Webpack is not really needed for this project. I'll have the same node modules and library as yours once I clone your online repository.

Please check if the app is running correctly on your local repository, if it does then you probably need to update your code on the online repository with your local repository

Reply:
Hello @rloterh.
Thank you very much for your suggestion.
I tested the project and it is running correctly on my local repository. I also remove and created again package and package-lock json files. But I saw It doesn't work on the remote repository.
I don't know what other thing can I do.
Thank you.